File: CS/MyLayoutView/MyLayoutView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.Layout;
using DevExpress.XtraGrid;

namespace MyXtraGrid
{
    public class MyLayoutView : LayoutView
    {
        public MyLayoutView()
        {
            
        }
        public MyLayoutView(DevExpress.XtraGrid.GridControl ownerGrid)
            : base(ownerGrid)
        {
            // put your initialization code here
        }
        protected override string ViewName { get { return "MyLayoutView"; } }
    }
}

File: VB/MyLayoutView/MyLayoutView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.Layout
Imports DevExpress.XtraGrid

Namespace MyXtraGrid
	Public Class MyLayoutView
		Inherits LayoutView
		Public Sub New()

		End Sub
		Public Sub New(ByVal ownerGrid As DevExpress.XtraGrid.GridControl)
			MyBase.New(ownerGrid)
			' put your initialization code here
		End Sub
		Protected Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyLayoutView"
			End Get
		End Property
	End Class
End Namespace
